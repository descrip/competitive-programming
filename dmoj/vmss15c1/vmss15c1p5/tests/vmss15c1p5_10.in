8 832 932
email 379 29 29
ermergerd 635 29 3
jemand 958 26 8
knapsack 885 25 9
partimejob 841 19 11
rhode 937 6 12
rotten 595 4 20
www 814 1 6
