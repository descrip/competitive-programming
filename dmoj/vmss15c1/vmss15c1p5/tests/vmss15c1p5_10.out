131330
email 0
ermergerd 22
jemand 1
knapsack 0
partimejob 0
rhode 0
rotten 0
www 143
